python/taichi/core/util.py

```diff
@@ -113,13 +113,39 @@ def format_plain_text(fn):
         f.write(formatted)
 
 
+def _find_clang_format_bin():
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    ENDC = '\033[0m'
+
+    clang_format_candidates = ['clang-format-6.0', 'clang-format']
```

Comment:
I don't have windows machine, but does `clang-format` on Win ends with `.exe`..?

Reply:
No need, when typing gcc, cmd will automatically search for `gcc.exe`, `gcc.bat`, `gcc.cmd`, `gcc.vbs`... in PATH.

Reply:
cool thx!